config/schedule.rb

```diff
@@ -7,3 +7,13 @@
 every :day, at: '3:00 am', roles: [:job_creator] do
   rake 'clear_expired_sessions'
 end
+
+# TODO: does Figaro work here?
+every ENV['queue_health_check_frequency_seconds'].to_i, roles: [:job_creator] do
+  runner 'WorkerHealthChecker.check'
+end
+
+# TODO: does Figaro work here?
+every ENV['queue_health_check_frequency_seconds'].to_i, roles: [:job_creator] do
+  runner 'WorkerHealthChecker.enqueue_dummy_jobs'
+end
```

Comment:
Ok, cool, I wasn't sure if the app environment was loaded for this file, or if it was run inside capistrano only, will switch to Figaro